app/agents/voice/automatic/processors/llm_spy.py

```
@@ -284,6 +285,18 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
 
             if self._enable_charts:
+                # Check if tool result contains isActionable flag
+                try:
+                    if isinstance(frame.result, dict) and frame.result.get(
+                        "isActionable", False
+                    ):
+                        mark_actionable_operation(self._session_id)
+                        logger.debug(
+                            f"[{self._session_id}] Marked actionable operation for tool: {frame.function_name}"
+                        )
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)